<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>waypointDialog</class>
 <widget class="QDialog" name="waypointDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>644</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>265</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wegmarkendialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Linux Libertine O</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Wegmarken-Eigenschaften:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Titel:</string>
       </property>
       <property name="buddy">
        <cstring>title</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="title"/>
     </item>
     <item>
      <widget class="QToolButton" name="nomatimButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Fragt einen Nominatim-Server nach dem Ort, der hier als Titel eingetragen ist. &lt;/p&gt;&lt;p&gt;Die Koordinaten des Punktes werden gemäß der Anfrage gesetzt.&lt;/p&gt;&lt;p&gt;Ggf werden weitere Waypoints erzeugt, wenn die Anfrage mehrere Ergebnisse lieferte.&lt;/p&gt;&lt;p&gt;Mehr dazu im Hilfetext zu Waypoints (Taste F1)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Bes&amp;chreibung</string>
     </property>
     <property name="buddy">
      <cstring>description</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="description">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>67</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="koordEing" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="kompassButton">
         <property name="text">
          <string>Kom&amp;pass</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Überträgt die Daten an den Atlas.&lt;/p&gt;&lt;p&gt;Das geschieht aber auch bei jeder Änderung eines Feldes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>&amp;Ok</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>S&amp;chließen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>löschen</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="centerButton">
       <property name="toolTip">
        <string>sorgt dafür, dass diese Wegmarke ins Anzeigefenster kommt. Brauchbar vor allem wenn man den Zoomfaktor verändert hat.</string>
       </property>
       <property name="text">
        <string>Wegmarke &amp;zentrieren</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="luftlinieButton">
       <property name="toolTip">
        <string>fragt nach einem weiteren Waypoint und berechnet einen Track, der die kürzeste Verbindung zwischen diesen Punkten darstellt.</string>
       </property>
       <property name="text">
        <string>Luftlinie zu...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="kachelKoordinatenBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>240</height>
      </size>
     </property>
     <property name="title">
      <string notr="true">Kachelkoordinaten</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="textLabel1_2">
          <property name="text">
           <string>&amp;Zoomstufe</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>Zoom</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="Zoom">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>18</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="Q3GroupBox" name="groupBox5_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>120</height>
           </size>
          </property>
          <property name="title">
           <string>Kacheldaten</string>
          </property>
          <widget class="QLabel" name="textLabel3_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>70</y>
             <width>50</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Pixel x</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QLabel" name="textLabel3_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>90</y>
             <width>40</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Pixel y</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QLabel" name="textLabel2_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>53</width>
             <height>15</height>
            </rect>
           </property>
           <property name="text">
            <string>Kachel x</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QLabel" name="textLabel2_2_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>53</width>
             <height>15</height>
            </rect>
           </property>
           <property name="text">
            <string>Kachel y</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="tile_y">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>40</y>
             <width>101</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="pix_x">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>70</y>
             <width>50</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>128</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="pix_y">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>90</y>
             <width>50</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>128</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="tile_x">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>20</y>
             <width>101</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="posCalcButton">
            <property name="minimumSize">
             <size>
              <width>156</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Position berechnen</string>
            </property>
            <property name="shortcut">
             <string>Alt+P</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="tileCalcButton">
            <property name="text">
             <string>&amp;Kacheldaten berechnen</string>
            </property>
            <property name="shortcut">
             <string>Alt+K</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="toggleButton">
       <property name="text">
        <string>^-----------^-Kachelkoordinaten ausblenden-^---------^</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>title</tabstop>
  <tabstop>description</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>kompassButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>Zoom</tabstop>
  <tabstop>tile_x</tabstop>
  <tabstop>tile_y</tabstop>
  <tabstop>pix_x</tabstop>
  <tabstop>pix_y</tabstop>
  <tabstop>toggleButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>calcTileData()</slot>
  <slot>calcPosition()</slot>
  <slot>hideAndShow()</slot>
 </slots>
</ui>
